stdlib/public/core/DebuggerSupport.swift

```diff
@@ -12,6 +12,40 @@
 
 import SwiftShims
 
+#if $Macros && hasAttribute(attached)
```

Comment:
I don't think you need this `#if` nowadays

Reply:
It seems it was needed? https://github.com/apple/swift/pull/69626#issuecomment-1848221971